samples/csharp_dotnetcore/51.teams-messaging-extensions-action/Pages/Shared/_Layout.cshtml

```diff
@@ -0,0 +1,20 @@
+﻿<!DOCTYPE html>
+<html lang="en">
+<head>
+    <title>Microsoft Teams Hello World Sample App</title>
+    <link rel="stylesheet" href="~/css/msteams-16.css" />
```

Comment:
.css files are not included?

![image](https://user-images.githubusercontent.com/11055362/121607634-3ce1a480-ca05-11eb-82d1-5a0538895299.png)